cmd/server/setup.go

```diff
@@ -82,22 +100,25 @@ func fallbackSqlite3File(path string) (string, error) {
 	}
 
 	// file is in new folder but not renamed
-	_, err = os.Stat(defaultDir)
+	_, err = os.Stat(dockerDefaultDir)
 	if err != nil && !os.IsNotExist(err) {
 		return "", err
 	}
 	if err == nil {
 		// rename in same folder should be fine as it should be same docker volume
-		return defaultPath, os.Rename(defaultDir, defaultPath)
+		log.Warn().Msgf("found sqlite3 file at '%s', rename to '%s'", dockerDefaultDir, dockerDefaultPath)
+		return dockerDefaultPath, os.Rename(dockerDefaultDir, dockerDefaultPath)
 	}
 
 	// file is still at old location
-	_, err = os.Stat(oldPath)
+	_, err = os.Stat(dockerOldPath)
 	if err == nil {
-		return oldPath, nil
+		log.Error().Msgf("found sqlite3 file at deprecated path '%s', please migrate to '%s'", dockerOldPath, dockerDefaultPath)
```

Comment:
Could we exit the server here?

Reply:
```suggestion
		log.Error().Msgf("found sqlite3 file at deprecated path '%s', please move it to '%s' and update your volume path if necessary", dockerOldPath, dockerDefaultPath)
```

Reply:
> Could we exit the server here?

@6543 What do you think about existing / stopping the server if the user has a database at the old docker path?

Reply:
I would do that in the next version and remove fallback in v0.17

Reply:
ajusted comment